ui/storybook/README.md

```diff
@@ -81,12 +79,55 @@ Below are some notable addons this project currently uses.
 - [Docs](https://storybook.js.org/addons/@storybook/addon-docs/) - Document component usage and properties in Markdown.
 - [Links](https://storybook.js.org/addons/@storybook/addon-links/) - Use to create links that navigate between stories.
 - [Measure](https://storybook.js.org/addons/@storybook/addon-measure/) - Helpful for inspecting layouts by visualizing the box model.
+- [Mock Service Worker](https://storybook.js.org/addons/msw-storybook-addon) - Allows mocking of api calls using [Mock Service Worker](https://mswjs.io/).
```

Comment:
Loving how you set up msw, I’ve been using it for perf testing using that mockTimeSeriesResponseWithManySeries function